vendorconsent/tcf1/rangesection.go

```diff
@@ -122,7 +122,7 @@ type rangeSection struct {
 }
 
 // VendorConsents implementation
-func (p rangeSection) VendorConsent(id uint16) bool {
+func (p rangeSection) VendorConsent(id uint16) bool { // TODO check if possible convert to pointer receiver
```

Comment:
We can investigate in the future. Not sure if the extra heap allocation would give us a speed boost. The `rangeSection` type is small... it's just 2 slice pointers and a bool.